MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include README
+include README.md
 include LICENSE
```

Comment:
Could you please split this one out into a separate commit?

Reply:
> 
> 
> Could you please split this one out into a separate commit?

Sure

Reply:
@eeshangarg #50 is ready